src/static/commandline.html

```diff
@@ -2,10 +2,12 @@
     <head>
         <meta charset="utf-8">
         <link rel="stylesheet" href="commandline.css">
+        <link rel="stylesheet" href="commandline-default.css">
+        <link rel="stylesheet" href="commandline-dark.css">
     </head>
     <body>
         <div id="completions"></div>
-        <input id=tridactyl-input></input>
+        <input id="tridactyl-input"></input>
         <script src="../commandline_frame.js"></script>
     </body>
 </html>
```

Comment:
Why have you changed the mode of userChrome-minimal.css?

Reply:
It was the only css that had 755, all others were 644, that looked like an accident to me. Is there a reason for the differing permissions?

Reply:
There's no reason for the permissions to continue to differ.